ops/model.py

```diff
@@ -634,3 +634,6 @@ def action_log(self, message):
 
     def action_fail(self, message=''):
         self._run(f'action-fail', f"{message}")
+
+    def juju_log(self, message, level):
+        self._run('juju-log', '--log-level', level, message)
```

Comment:
I'd like to see a test_model.py change associated with this, just focused on the Model object.

Reply:
Done:
https://github.com/canonical/operator/pull/133/commits/167a57b7e2544aed30002527dcd50bc56c8c2ada#diff-2bdcc3da19a2e638a126aab53c420862R819-R832